transport/grpc/config.go

```diff
@@ -103,7 +141,26 @@ type OutboundConfig struct {
 	yarpcconfig.PeerChooser
 
 	// Address to connect to if no peer options set.
-	Address string `config:"address,interpolate"`
+	Address string            `config:"address,interpolate"`
+	TLS     OutboundTLSConfig `config:"tls,optional"`
+}
+
+func (c OutboundConfig) dialOptions() []DialOption {
+	return c.TLS.dialOptions()
+}
+
+// OutboundTLSConfig configures TLS for a gRPC outbound.
+type OutboundTLSConfig struct {
+	Enabled bool `config:"enabled,optional"`
+}
+
+func (c OutboundTLSConfig) dialOptions() []DialOption {
+	if c.Enabled {
+		creds := credentials.NewClientTLSFromCert(nil, "")
+		option := WithTransportCredentials(creds)
```

Comment:
nit: invert if condition to reduce nesting

```
if !c.Enabled { return nil }
 creds := ..
option := ...
return []DialOption{option}
```

Reply:
done